stroeer/page/section/v1/section_page_service.proto

```diff
@@ -71,12 +71,25 @@ service SectionPageService {
 // Correct paths have a leading and a trailing slash, like `/nachrichten/unterhaltung/`
 // The homepage has the path `/`.
 message GetSectionPageRequest {
+  // valid section_path, with leading and trailing slash
   string section_path = 1;
+
+  // use to page through sections. If unspecified, it will default to `1`.
+  // Paging is 1-based (1 is the first page, there is no page `0`)
+  //
+  // Due to underlying mechanisms and seo requirements, page-size is fixed at 30
```

Comment:
not sure if this is going to be `30` in all cases (e.g. current regional sections have a page size of `10`). Maybe we document that the page size is fixed in the backend here.

Reply:
**POs hate this trick**: as long as its dividable, we could absolutely do this:

given internal pagination has a `page_size=30`, for the consumer we do the following to fake a page size of 10 by dividing our pages into 3 virtual pages:

1. `total_pages = 3 * total_pages`
2. `page = page / 3`
3. `skip = ((page + 2) % 3) * 10`
4. `articles = ddb.query(page).items().skip(skip).take(10)`